.storybook/preview.tsx

```diff
@@ -12,6 +12,13 @@ import Page from './components/Page'
 import { globalStyles } from './components/globalStyle'
 import { dark, light } from './storybookThemes'
 import '@ultraviolet/fonts/fonts.css'
+import { useEffect } from 'react'
+import { scan } from "react-scan"
+
+scan({
```

Comment:
normal les traces de react scan dans cette MR ?

Reply:
it needs a small rebase let me check